<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowBook</class>
 <widget class="QMainWindow" name="MainWindowBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>619</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Mangle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="widget_list_files">
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="col2">
      <item>
       <widget class="QGroupBox" name="title_block">
        <property name="title">
         <string>Book</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_line">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkbox_webtoon">
           <property name="text">
            <string>Is a Webtoon (split in N images)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="export_block">
        <property name="title">
         <string>Export</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="device_block">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="device_label">
             <property name="text">
              <string>Device</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="device_select">
             <item>
              <property name="text">
               <string>Kindle 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kindle 2/3/Touch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kindle 4 &amp; 5</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kindle DX/DXG</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kindle Paperwhite 1 &amp; 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kindle Paperwhite 3/Voyage/Oasis</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Mini/Touch</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Glo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Glo HD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Aura</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Aura HD</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Aura H2O</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Libra H2O</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kobo Elipsa 2E</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkbox_split_right_left">
           <property name="text">
            <string>Split images into two pages (right, left)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkbox_split_left_right">
           <property name="text">
            <string>Split images into two pages (left, right)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="col3">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QGroupBox" name="export_block2">
        <property name="maximumSize">
         <size>
          <width>2000</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Export</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_23">
         <item>
          <widget class="QPushButton" name="export_button">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="export_progress_bar">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew"/>
    <addaction name="separator"/>
    <addaction name="actionFileExit"/>
   </widget>
   <widget class="QMenu" name="menu_Book">
    <property name="title">
     <string>&amp;Book</string>
    </property>
    <addaction name="actionBookOptions"/>
    <addaction name="separator"/>
    <addaction name="actionBookExport"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpHomepage"/>
    <addaction name="actionHelpAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Book"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileNew"/>
   <addaction name="separator"/>
   <addaction name="actionBookRemove"/>
   <addaction name="actionBookShiftUp"/>
   <addaction name="actionBookShiftDown"/>
   <addaction name="separator"/>
   <addaction name="actionBookExport"/>
  </widget>
  <action name="actionFileNew">
   <property name="icon">
    <iconset>
     <normaloff>../img/file_new.png</normaloff>../img/file_new.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>New book</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset>
     <normaloff>../img/file_open.png</normaloff>../img/file_open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="toolTip">
    <string>Open book</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset>
     <normaloff>../img/save_file.png</normaloff>../img/save_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save book</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
   <property name="toolTip">
    <string>Save book as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBookOptions">
   <property name="icon">
    <iconset>
     <normaloff>../img/book.png</normaloff>../img/book.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Options...</string>
   </property>
  </action>
  <action name="actionBookRemove">
   <property name="icon">
    <iconset>
     <normaloff>../img/remove_files.png</normaloff>../img/remove_files.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove files</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBookExport">
   <property name="icon">
    <iconset>
     <normaloff>../img/export_book.png</normaloff>../img/export_book.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Export...</string>
   </property>
   <property name="toolTip">
    <string>Export book</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionHelpHomepage">
   <property name="text">
    <string>&amp;Homepage...</string>
   </property>
  </action>
  <action name="actionHelpAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionBookAddFiles">
   <property name="icon">
    <iconset>
     <normaloff>../img/add_file.png</normaloff>../img/add_file.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Files...</string>
   </property>
   <property name="toolTip">
    <string>Add files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBookAddDirectory">
   <property name="icon">
    <iconset>
     <normaloff>../img/add_directory.png</normaloff>../img/add_directory.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Directory...</string>
   </property>
   <property name="toolTip">
    <string>Add directory</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBookShiftUp">
   <property name="icon">
    <iconset>
     <normaloff>../img/shift_up.png</normaloff>../img/shift_up.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Up</string>
   </property>
   <property name="toolTip">
    <string>Shift files up</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBookShiftDown">
   <property name="icon">
    <iconset>
     <normaloff>../img/shift_down.png</normaloff>../img/shift_down.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Down</string>
   </property>
   <property name="toolTip">
    <string>Shift files down</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionFileExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowBook</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
